apps/docs/content/guides/platform/http-status-codes.mdx

```diff
@@ -18,6 +18,17 @@ The Supabase platform offers several HTTP APIs for each project. These APIs can
 
 4XX status codes indicate an issue on the client's end with the request being made. These could include missing or invalid auth information, a malformed request, making too many requests in too short a time period ("rate limiting"), or a network issue on the client's end.
 
+### 402 service restriction
+
+In case the Fair Use Policy is applied, the projects of your organization may be restricted from processing requests. If service restrictions are applied, projects will return a 402 status code together with a description on why the project is restricted. The description can contain one or more of the following codes:
```

Comment:
```suggestion
In case the Fair Use Policy is applied, the projects of your organization may be restricted from processing requests. If service restrictions are applied, projects will return a 402 status code together with a description of why the project is restricted. The description can contain one or more of the following codes:
```